repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/FastScanner.java

```diff
@@ -46,6 +46,7 @@ public void run() {
         int nInteresting = 0;
         Set<Long> done = new HashSet<>();
         JobHelperv2 jobHelperv2 = new JobHelperv2(RepairnatorConfig.getInstance().getJTravis());
+        SequencerCollector collector = new SequencerCollector();
```

Comment:
a little but of inheritance would help: this should be in a subclass of FastScanner `SequencerLearnerScanner`

Reply:
WDT about a Scanner abstract class?

Reply:
Since the scanners do not share a common interface I think it is ok to just leave both implementing runnable and not creating an abstract class just because. I think your point was for me to not modify FastScanner directly. Am I correct?